mariadb-demo/dags/deploy-maria-cluster.yaml

```diff
@@ -0,0 +1,23 @@
+name: Deploy mariadb cluster
+description: deploy mariadb cluster with prometheus and grafana on the grid
+tags: demo
+
+env:
+  - MNEMONICS: ${MNEMONICS}
+  - NETWORK: ${NETWORK}
+
+steps:
+  - name: validate
+    command: terraform version
+
+  - name: init
+    command: terraform init
+    dir: tf
```

Comment:
I think we need to add the relative path to the parent dir of the repo

Reply:
it is relative from the point you pass to `—dags` while running the server